invokeai/frontend/.yarnrc.yml

```diff
@@ -0,0 +1 @@
+nodeLinker: node-modules
```

Comment:
I guess this is a yarn v2+ file?

Reply:
Yes. The newest revision should clear some of the confusion as it utilizes a specific yarn version to standardize whichever the developer is utilizing. It'll still include this file, but only as much to relate it back to the specific version of yarn the project needs to run without getting into trouble.